Sources/Extensions/UIKit/UIApplicationExtension.swift

```diff
@@ -7,19 +7,19 @@ import UIKit
 
 public extension UIApplication {
 
-    public func topViewController(baseViewController: UIViewController? = UIApplication.shared.keyWindow?.rootViewController) -> UIViewController? {
-        guard let base = baseViewController else {
+    public func topViewController(_ baseVC: UIViewController? = UIApplication.shared.keyWindow?.rootViewController) -> UIViewController? {
```

Comment:
I think the argument label should not be omitted. Maybe add a preposition? I don't know very much about the terminologies but maybe `from` or `on`?